src/myr/Myr.js

```diff
@@ -421,7 +427,25 @@ class Myr {
         return this.cursor.color;
     }
 
-    getRandomColor = (colors = null) => {
+    setTexture = (texture = "blank") => {
+        let holdKey;
+
+        TexturePack.forEach(function(item) {
+            Object.keys(item).forEach(function(key) {
+                if(key === texture){
+                    holdKey = item[key];
+                }
+            });
+        });
+        if(holdKey){
+            this.cursor.texture = holdKey;
+        } else {
+            this.cursor.texture = texture;
```

Comment:
We should have at least some form of basic regex check to make sure the URL is valid.

Reply:
There should be some form of error handling here if it's invalid.